addons/l10n_tw_edi_ecpay/models/account_move.py

```diff
@@ -0,0 +1,280 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import datetime
+import uuid
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.addons.l10n_tw_edi_ecpay.utils import EcPayAPI
+
+
+class AccountMove(models.Model):
```

Comment:
Your methods could use more docstrings to explain their purpose and provide insight into the API being used.
Don't forget that in the future, other developers will work on this, so try to help them as much as possible.